### ISSUE TITLE
Warning to sort out

### ISSUE THREAD

**Issue by annecori** [reconhub/vimes#5]
``` r
checking R code for possible problems ... NOTE
cutoff.choice.interactive: no visible global function definition for
  ‘abline’
cutoff.choice.interactive: no visible global function definition for
  ‘plot’
Undefined global functions or variables:
  abline plot
Consider adding
  importFrom("graphics", "abline", "plot")
to your NAMESPACE file.
```

**Comment by thibautjombart** [comment]
OK got it thanks

## 

Dr Thibaut Jombart
Lecturer, Department of Infectious Disease Epidemiology
Imperial College London
https://sites.google.com/site/thibautjombart/
https://github.com/thibautjombart
Twitter: @TeebzR https://twitter.com/TeebzR

**Comment by thibautjombart** [comment]
Fixed as of 398a4f2
